packages/desktop-gui/src/lib/markdown-renderer.jsx

```diff
@@ -26,10 +26,22 @@ export default class MarkdownRenderer extends React.PureComponent {
   }
 
   render () {
+    let html = md.render(this.props.markdown)
+
+    // markdown-it wraps the markdown in a <p> tag, this removes it to prevent
+    // styling issues
+    if (this.props.noParagraphWrapper) {
+      const matches = /^<p>([^<]*)<\/p>\n$/.exec(html)
+
+      if (matches) {
+        html = matches[1]
+      }
+    }
```

Comment:
I think you can achieve the same thing with `md.renderInline()`.

Reply:
nice, i flew right by that function when i skimmed the markdown-it README :man_facepalming: